Net/src/DNS.cpp

```diff
@@ -132,6 +141,13 @@ HostEntry DNS::hostByAddress(const IPAddress& address, unsigned
 			freeaddrinfo(pAI);
 			return result;
 		}
+#if defined(_WIN32)
+		else if(rc == WSANO_DATA && address.isLoopback()) // no data record found and is local host 
```

Comment:
Does it make sense putting this condition first, before checking the "localhost" hostname?